### PR TITLE
Doc fix

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

File: Documentation/dev/developer-guide.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ $ pip3 install operator-courier
 
 It is important that you clone the project to the right location in your $GOPATH.
 
-The repository much be located at:
+The repository path should look like this:
```
Comment (pruan-rht) [inline Documentation/dev/developer-guide.md]: I think that was just a typo... how about just subbing `much` with `must`

Comment (timflannagan1) [reply Documentation/dev/developer-guide.md]: +1

Comment (EmilyM1) [reply Documentation/dev/developer-guide.md]: Oh is that what it was. Sure.

**Pull request merged**